sagemaker-autopilot/housing_prices/autopilot_california_housing.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.    # if os.path.exists("SageMakerAutopilotDataExplorationNotebook.ipynb"):
</pre><p>Can we wrap this into a utility, pass filename as a parameter to it, and instead of commented code here can we move this into a script, whose invocation is commented out instead? I feel it will make the notebook cleaner.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
I uncommented the code as you suggested in another comment. Do you still feel it would be better to wrap this snippet into a utility?